<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2052</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>1571</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>进程管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>1501</width>
        <height>821</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_query">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_query">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>170</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_create">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>30</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>30</y>
        <width>170</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>创建</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>使用率</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>1451</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>1451</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>660</y>
        <width>1451</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 60 12pt &quot;Ubuntu Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>C
P
U
使用率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>51</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 60 12pt &quot;Ubuntu Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>内存使用率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>640</y>
        <width>51</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 60 12pt &quot;Ubuntu Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>交换分区使用率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>主机名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>系统版本号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_launch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>持续运行时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cpu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU型号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cpu_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU主频:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_host">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="line_version">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_launch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_run">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_cpu">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="line_cpu_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>900</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shut">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>1070</y>
      <width>181</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>结束进程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>1070</y>
      <width>170</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_shutdown">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>1080</y>
      <width>471</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2052</width>
     <height>49</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
